Makefile

```diff
@@ -1407,6 +1407,28 @@ lint-py:
 	$(warning Run 'make lint-py-build')
 endif
 
+.PHONY: lint-yaml-build
+# python -m pip install yamllint
+# Try with '--system' if it fails without; the system may have set '--user'
+# Jenkins CI lint job uses python2 for linting. When Node.js 12.x is EOL, we
+# can remove that from the Jenkins job config and $(PYTHON) for YAML linting
+# just like everywhere else. (yamllint does not support Python 2.)
+lint-yaml-build:
+	$(info Pip installing yamllint on $(shell $(PYTHON) --version)...)
+	python3 -m pip install --upgrade -t tools/pip/site-packages yamllint || \
+		python3 -m pip install --upgrade --system -t tools/pip/site-packages yamllint
+
+ifneq ("","$(wildcard tools/pip/site-packages/yamllint)")
+.PHONY: lint-yaml
+# Lints the YAML files with yamllint.
+lint-yaml:
+	PYTHONPATH=tools/pip python3 -m yamllint .
```

Comment:
```suggestion
	PYTHONPATH=tools/pip $(PYTHON) -m yamllint .
```

Reply:
See comment starting on line 1413 for the explanation as to why this is (at least temporarily) like it is now rather than the suggestion here.